jetty-http/src/main/java/org/eclipse/jetty/http/compression/NBitStringParser.java

```diff
@@ -11,15 +11,17 @@
 // ========================================================================
 //
 
-package org.eclipse.jetty.http3.qpack.internal.util;
+package org.eclipse.jetty.http.compression;
 
 import java.nio.ByteBuffer;
 
+import org.eclipse.jetty.util.CharsetStringBuilder;
+
 public class NBitStringParser
```

Comment:
Can you take the opportunity to add a bit of javadoc to these classes

Reply:
done